exercises/rust-2-a/README.md

```diff
@@ -0,0 +1,56 @@
+## Introduction
+
+This is Part 2 of our research into how people write Rust in varying styles. This exercise should take 10-15 minutes. There is no right or wrong way to approach it - just do what feels the most natural to you. Feel free to use books, Google or Stack Overflow, just like you would if you were programming normally, but please don't look for a complete solution to the problem as this will negatively affect the research.
+
+## Instructions
```

Comment:
Maybe the instruction could be a little bit more explicit about the student being given some existing, working code? As an example, consider the go-2-a exercise's [instructions](https://github.com/exercism/research_experiment_1/blob/master/exercises/go-2-a/README.md#instructions).